src/main/java/atreides/task/Task.java

```diff
@@ -0,0 +1,57 @@
+package atreides.task;
+
+/**
+ * represents the tasks that the user can add into the list
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task() {
+        this("");
+    }
+
+    public Task(String description) {
+        this(description, false);
+    }
+
+    public Task(String description, boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+    /**
+     * represents whether the task has been completed or not
+     * @param trueString
+     * @param falseString
+     * @return trueString if completed else falseString
```

Comment:
Should you add a description for these params to avoid confusion for what they represent?